client/src/components/form-inputs/MultiSelect.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { FormOptionValue } from '@konomi.ai/c-form';
+import {
+  Box,
+  Chip,
+  FormControl,
+  ListItemText,
+  ListItemIcon,
+} from '@mui/material';
+import CheckIcon from '@mui/icons-material/Check';
+import CloseIcon from '@mui/icons-material/Close';
+import {
+  Control,
+  Controller,
+  FieldValues,
+  UseFormSetValue,
+} from 'react-hook-form';
+import { isNonEmptyArrayOfStrings } from '../../helpers';
+import { StyledInputLabel, StyledSelect, StyledMenuItem } from './styled';
+
+export interface ISelectProps {
+  control: Control;
+  setValue: UseFormSetValue<FieldValues>;
+  name: string;
+  label: string;
+  required?: boolean;
+  options: FormOptionValue[];
+}
+
+export const MultiSelect = ({
```

Comment:
Component should support passing children as options instead of just a data array.

Also convert to `React.FC`